.docker-compose.datagov-theme.yaml

```diff
@@ -25,8 +27,8 @@ services:
           install_standard_ckan_extension_github -r ckan/ckanext-harvest &&\
           install_standard_ckan_extension_github -r datopian/ckanext-logstash &&\
           install_standard_ckan_extension_github -r ckan/ckanext-dcat ckanext-dcat &&\
-          install_standard_ckan_extension_github -r GSA/ckanext-geodatagov &&\
-          install_standard_ckan_extension_github -r akariv/ckanext-datajson -b datagov &&\
+          install_standard_ckan_extension_github -r datopian/ckanext-geodatagov &&\
+          install_standard_ckan_extension_github -r avdata99/ckanext-datajson -b datagov &&\
```

Comment:
Shouldn't these be the GSA repos?

Reply:
> Shouldn't these be the GSA repos?
I think they can be repos at GSA. I already sent a PR for GeoDataGov: https://github.com/GSA/ckanext-geodatagov/pull/120